.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+branch = true
+source = 
+  pytest_voluptuous
+  tests
+  
```

Comment:
Let's move this config to `setup.cfg`. I want to keep the source tree free of extra files, and `setup.cfg` is already there so let's re-use it.

```ini
[coverage:run]
branch = true
source = pytest_voluptuous
```

Also add:

```ini
[coverage:report]
show_missing = 1
```

So we can reproduce `--cov-report term-missing` from `pytest-cov`.